src/main/java/com/hivemq/extensions/configuration/DnsDiscoveryConfigExtended.java

```diff
@@ -59,6 +61,48 @@ public String discoveryAddress() {
         return discoveryAddress;
     }
 
+    /**
+     * method to get initial discovery interval
+     * either from environment variable
+     * or from properties configuration
+     * or default setting
+     *
+     * @return int - the initial discovery interval
+     */
+    public int initialDiscoveryInterval() {
+        final @Nullable String initialDiscoveryInterval = System.getenv(INITIAL_DISCOVERY_INTERVAL_ENV);
+
+        if (initialDiscoveryInterval != null && !initialDiscoveryInterval.isEmpty()) {
+            try {
+                return Integer.parseInt(initialDiscoveryInterval);
+            } catch (NumberFormatException e) {
+                log.error("Resolution timeout from env {} couldn't be parsed to int. Fallback to config value.", INITIAL_DISCOVERY_INTERVAL_ENV);
```

Comment:
The log was not adapted, still uses resolution timeout instead of initial discovery interval